plugins/thehive/bin/komand_thehive

```diff
@@ -1,40 +1,54 @@
 #!/usr/bin/env python
 # GENERATED BY KOMAND SDK - DO NOT EDIT
-import komand
-from komand_thehive import connection, actions, triggers
-
+import os
+import json
+from sys import argv
 
 Name = "TheHive"
 Vendor = "rapid7"
 Version = "2.0.5"
 Description = "TheHive is a scalable, open source security incident response solution designed for SOCs & CERTs to collaborate, elaborate, analyze and get their job done. Handle your case management needs with TheHive plugin for Rapid7 InsightConnect"
 
 
-class ICONThehive(komand.Plugin):
-    def __init__(self):
-        super(self.__class__, self).__init__(
+def main():
+    if 'http' in argv:
+        if os.environ.get("GUNICORN_CONFIG_FILE"):
+            with open(os.environ.get("GUNICORN_CONFIG_FILE")) as gf:
+                gunicorn_cfg = json.load(gf)
+                if gunicorn_cfg.get("worker_class", "sync") == "gevent":
+                    from gevent import monkey
+                    monkey.patch_all()
+        elif 'gevent' in argv:
+            from gevent import monkey
+            monkey.patch_all()
+
+    import komand
+    from komand_thehive import connection, actions, triggers
```

Comment:
Note to self whats going on here?
Is this a bug with `insight-plugin refresh` when refreshing a `komand_{plugin}`?